tor/core/inbox.py

```diff
@@ -172,23 +165,11 @@ def check_inbox(config):
         elif item.subject in ('comment reply', 'post reply'):
             process_reply(item, config)
 
-        elif 'reload' in item.subject.lower():
+        elif item.subject[0] == '!':
+            # Handle our special commands
+            process_command(item, config)
             item.mark_read()
```

Comment:
Nice! I like that this is left in there to mark the task as done, that way we don't keep re-parsing it if there's a bug in the admin command not marking it as read.

Reply:
Very nice 👍